docs/components/gripper/softrobotics.md

```diff
@@ -0,0 +1,87 @@
+
+{{< tabs name="Configure a softrobotics gripper" >}}
+{{% tab name="Config Builder" %}}
+
+On the **COMPONENTS** sub-tab of your robot's page in the [Viam app](https://app.viam.com/), navigate to the **Create Component** menu.
+Enter a name for your gripper, select the type `gripper`, and select the `softrobotics` model.
+
+![Creation of a softrobotics gripper component in the Viam app config builder.](../../img/gripper/softrobotics-builder.png)
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "name": <gripper_name>,
+            "type": "gripper",
+            "model" : "softrobotics",
+            "attributes": {
+                "board": <board_name>,
+                "open": <pin_number>,
+                "close": <pin_number>,
+                "power": <pin_number>,
+                "analog_reader": "psi"
+            }
+        }
+    ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "name": "local",
+            "type": "board",
+            "model": "pi",
+            "attributes": {
+                "analogs": [
+                    {
+                        "name": "psi",
+                        "pin": "32"
+                    }
+                ]
+            }
+        },
+        {
+            "name": "my_gripper",
+            "type": "gripper",
+            "model" : "softrobotics",
+            "attributes": {
+                "board": "local",
+                "open": "11",
+                "close": "13",
+                "power": "15",
+                "analog_reader": "psi"
+            }
+        }
+    ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `softrobotics` grippers:
+
+Name | Inclusion | Type | Description
+---- | --------- | ---- | -----------
+`board` | **Required** | string | The name of the [board](../../board/) to which your gripper [control unit](https://www.softroboticsinc.com/uploads/2020/05/Tech_Sheet_coDrive_Control_Unit_-__TS-200210_Rev_B.pdf) is wired.
+`open` | **Required** | string | The pin number of the board pin wired to the open pin (D1) on the gripper controller.
+`close` | **Required** | string | The pin number of the board pin wired to the close pin (D2) on the gripper controller.
+`power` | **Required** | string | The pin number of the board pin wired to the enable pin (D3) on the gripper controller.
+`psi` | **Required** | string | Needs to be called `"psi"`, and you need to configure an analog named `"psi"` on your [board](../../board/).

```

Comment:
should this be `analog_reader` ?
and maybe link to /components/board/#analogs directly

Reply:
```suggestion
`analog_reader` | **Required** | string | Needs to be called `"psi"`, and you need to configure an [analog](../../board/#analogs) named `"psi"` on your board.
```